constraint/pkg/client/backend.go

```diff
@@ -54,6 +59,7 @@ func (b *Backend) NewClient(opts ...ClientOpt) (Client, error) {
 	if len(errs) > 0 {
 		return nil, errs
```

Comment:
Isn't this returning an array but expecting an `error`?

Reply:
The array obeys the Error interface